### PR TITLE
Define conditional statements

### LINKED ISSUE #180: Define conditionals
Clarify the prose for conditionals, and perhaps what should be considered true/false.

### PR DESCRIPTION
**Pull request by miketaylr**
Fixes #180

***
<a href="https://whatpr.org/infra/592.html" title="Last updated on Jul 12, 2023, 6:42 PM UTC (5fac617)">Preview</a> | <a href="https://whatpr.org/infra/592/dbc268d...5fac617.html" title="Last updated on Jul 12, 2023, 6:42 PM UTC (5fac617)">Diff</a>

### CODE REVIEW

**Comment by miketaylr** [comment]
Oops, fixed the typo. Ready for review @annevk.

**Comment by miketaylr** [comment]
Or maybe @domenic has time to review.

**Comment by domenic** [comment]
I gave it a shot but got stuck on https://github.com/whatwg/infra/issues/180#issuecomment-1639360076.

**Comment by miketaylr** [comment]
> I gave it a shot but got stuck on [#180 (comment)](https://github.com/whatwg/infra/issues/180#issuecomment-1639360076).

Sorry to rub your brain the wrong way! 🧠

**Comment by alecodesign** [comment]
> Fixes #180
> 
> 
> 
> ***
> <a href="https://whatpr.org/infra/592.html" title="Last updated on Jul 12, 2023, 6:42 PM UTC (5fac617)">Preview</a> | <a href="https://whatpr.org/infra/592/dbc268d...5fac617.html" title="Last updated on Jul 12, 2023, 6:42 PM UTC (5fac617)">Diff</a>

File: infra.bs [hunk 1]
```diff
@@ -557,6 +557,49 @@ the end result is indistinguishable. For instance, as long as |result| in the ab
...
+
+<div class=example id=example-conditional-statement-no-then-1>
+ <ol>
+  <li><p>Let |value| be null.
+
+  <li><p>If |input| is a <a>string</a>, then set |value| to |input|.
+
+  <li><p>Otherwise, set |value| to failure.
+
+  <li><p>Return |value|.
```
Comment (annevk) [inline infra.bs:584]: You could add a step before this that asserts that value is not null. It's a bit obvious here, but the suggestion might help people writing more complicated algorithms.

**Review by annevk — APPROVED** [review]
I'd be happy with landing this. It matches the style we use throughout.

Found one optional improvement and I guess @domenic should weigh in again given his earlier reservations.

**Review by domenic — APPROVED** [review]

**Pull request merged**